plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -653,23 +653,21 @@ private data class AdditionalData(
 )
 
 /**
- * Clean the [rawVersion] string contained in a Yarn 2+ locator to have it compatible with NPM/Semver.
+ * Clean this Yarn2 version string (originating from a Yarn 2+ locator) for compatibility with NPM/Semver.
  */
-private fun cleanYarn2VersionString(rawVersion: String): String {
-    // 'Patch' locators are complex expressions such as
-    // resolve@npm%3A2.0.0-next.3#~builtin<compat/resolve>%3A%3Aversion=2.0.0-next.3&hash=07638b
-    // Therefore, the version has to be extracted (here '2.0.0-next.3').
-    var result = rawVersion.substringAfter("version=")
+private fun String.cleanVersionString(): String =
+    this
```

Comment:
`this` is redundant, are you keeping it for formatting reasons?

Reply:
yes.

Reply:
Ok, I guess I'm fine with that.